packages/flutter/lib/src/material/slider.dart

```diff
@@ -208,8 +214,13 @@ class _SliderState extends State<Slider> with TickerProviderStateMixin {
   static const Duration enableAnimationDuration = const Duration(milliseconds: 75);
   static const Duration positionAnimationDuration = const Duration(milliseconds: 75);
 
+  /// Animation controller that is run when interactions occur (taps, drags,
+  /// etc.).
   AnimationController reactionController;
+  /// Animation controller that is run when enabling/disabling the slider.
   AnimationController enableController;
+  /// Animation controller that is run when transitioning between one value
+  /// and the next on a discrete slider.
   AnimationController positionController;
```

Comment:
those should probably be `//`

Reply:
It was an experiment to see if that would take care of the last three missing docs... :-)